psi4/src/export_fock.cc

```diff
@@ -67,9 +67,13 @@ void export_fock(py::module &m) {
         .def("set_do_K", &JK::set_do_K)
         .def("set_do_wK", &JK::set_do_wK)
         .def("set_omega", &JK::set_omega)
+        .def("get_omega", &JK::get_omega)
+        .def("set_wcombine", &JK::set_wcombine)
+        .def("get_wcombine", &JK::get_wcombine)
         .def("set_omega_alpha", &JK::set_omega_alpha, "Weight for HF exchange term in range-separated DFT", "alpha"_a)
+        .def("get_omega_alpha", &JK::get_omega)
         .def("set_omega_beta", &JK::set_omega_beta, "Weight for dampened exchange term in range-separated DFT", "beta"_a)
-        .def("set_wcombine", &JK::set_wcombine)
+        .def("get_omega_beta", &JK::get_omega)
```

Comment:
get_omega_alpha/beta need to have get_omega_alpha/beta as 2nd arg, not get_omega. may as well add docstring, too.

Reply:
Thank you. I'm sorry I let the typo slip in. I must have gotten carried away with vim tricks

Reply:
np. at least you didn't create a cubic black hole with ~radius 1 bohr (see #1938)